src/components/SessionList.vue

```diff
@@ -177,19 +183,18 @@ export default {
 
 		.icon-more, .icon-group, .icon-settings-dark {
 			background-color: var(--color-background-dark);
-			width: 36px;
-			height: 36px;
-			margin: 6px 6px 6px 0px;
+			width: 44px;
+			height: 44px;
+			margin: 0px 6px 0px 0px;
 		}
 	}
 
 	.avatar-wrapper {
-		width: 32px;
-		height: 32px;
+		width: 42px;
+		height: 42px;
```

Comment:
```suggestion
		width: 44px;
		height: 44px;
```
Otherwise it's not vertically aligned with `.avatardiv.icon-group`.